web/app/integration/specs/logo-redirect.js

```diff
@@ -0,0 +1,11 @@
+const assert = require('assert');
+  describe('logo link test', function() {
+      it('should redirect to the home view if logo is clicked', () => {
```

Comment:
this indentations seems a little weird. should it be

```js
  describe('logo link test', function() {
    it('should redirect to the home view if logo is clicked', () => {
      browser.url('http://localhost:7777/tap');
      const link = $('.linkerd-word-logo');
      link.click();
      const breadcrumbHeader = $('.breadcrumb-link');
      const pageTitle = breadcrumbHeader.getText();
      assert.equal(pageTitle, 'Overview');
    });
  });
```

Reply:
weird, thanks for the catch